frontend/src/plugin-extensions/demo-plugin/package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "demo-plugin",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "clean": "rm -rf dist",
+        "build-dev": "yarn clean && NODE_ENV=development yarn ts-node node_modules/.bin/webpack",
+        "start": "yarn clean && NODE_ENV=development yarn ts-node node_modules/.bin/webpack serve",
+        "http-server": "./http-server.sh dist",
+        "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c i18next-parser.config.js",
```

Comment:
we don't have any `locales` directory in this PR, but we have parser config and script defined...

Reply:
right now demo components dont have any translation text, but if anyone add in future they can use this command to generate locale

Reply:
in `CopyWebpackPlugin` we will be specifying path for locales directory, if there is no such directory will plugin still work without any error ?

Reply:
i added locales